Package.swift

```diff
@@ -1,32 +1,16 @@
-//
-//  Package.swift
-//  Kingfisher
-//
-//  Created by Wei Wang on 16/1/22.
-//
-//  Copyright (c) 2018 Wei Wang <onevcat@gmail.com>
-//
-//  Permission is hereby granted, free of charge, to any person obtaining a copy
-//  of this software and associated documentation files (the "Software"), to deal
-//  in the Software without restriction, including without limitation the rights
-//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-//  copies of the Software, and to permit persons to whom the Software is
-//  furnished to do so, subject to the following conditions:
-//
-//  The above copyright notice and this permission notice shall be included in
-//  all copies or substantial portions of the Software.
-//
-//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-//  THE SOFTWARE.
-
+// swift-tools-version:4.2
 import PackageDescription
 
 let package = Package(
-  name: "Kingfisher",
-  exclude: ["Tests", "Sources/AnimatedImageView.swift", "Sources/UIButton+Kingfisher.swift", "Sources/WKInterfaceImage+Kingfisher.swift"]
+    name: "Kingfisher",
+    // platforms: [.iOS("10.0"), .macOS("10.12"), tvOS("10.0"), .watchOS("3.0")],
+    products: [
+        .library(name: "Kingfisher", targets: ["Kingfisher"])
+    ],
+    targets: [
+        .target(
+            name: "Kingfisher",
+            path: "Sources"
+        )
```

Comment:
Should we also add the test target here? (Or is it possible to do so?)

Reply:
Adding the test target only makes sense in order to be able to run the `swift test` command which will currently not work due to the incompatibility of SwiftPM with UIKit AFAIK. Unless someone proves me wrong, it would only add a "maybe future support for SwiftPM" if we already added the test targets now.

Please also note that SwiftPM currently always checks out all dependencies listed under `dependencies` without looking at the targets they are needed for – therefore if you had dependencies that were only needed in the test targets, they would also be checked out in project where Kingfisher would be used without any purpose (since `swift test` doesn't work at the moment). Therefore I don't think the test targets should be added as of now.